merlin-framework-processor/src/main/java/gov/nasa/jpl/aerie/merlin/processor/generator/MapperMethodMaker.java

```diff
@@ -119,8 +118,24 @@ default MethodSpec makeGetArgumentsMethod(final ExportTypeRecord exportType) {
         .build();
   }
 
-  default MethodSpec makeGetValidationFailures(final ExportTypeRecord exportType) {
-    final var metaName = getMetaName(exportType);
+  public final MethodSpec makeGetRequiredParametersMethod() {
```

Comment:
I like the explicit final/not-final demarcations guiding the subclasses as to which methods to actually implement :+1: